frontend/css/style.css

```diff
@@ -1,28 +1,123 @@
+@import url('https://fonts.googleapis.com/css2?family=Comic+Neue:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap');
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 :root {
-    --text-color: #0A0707;
-    --bg-color: #FDFCFC;
-    --text-correct-color: #177915;
-    --primary-color: #903634;
-    --secondary-color: #C99E9E;
-    --accent-color : #7DD0F3;
-    --card-bg-color: #E2E8F0;
-    --select-color : #7DD0F3;
+  --text-color: #0a0707;
+  --bg-color: #fdfcfc;
+  --text-correct-color: #177915;
+  --primary-color: #903634;
+  --secondary-color: #c99e9e;
+  --accent-color: #7dd0f3;
+  --card-bg-color: #e2e8f0;
+  --select-color: #7dd0f3;
+}
+
+.dark-mode {
+  --text-color: #dbf6fd;
+  --bg-color: #181b1c;
+  --primary-color: #9e092a;
+  --secondary-color: #434a5a;
+  --accent-color: #a5cad7;
+  --card-bg-color: #202528;
+}
+
+html, body {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  color: var(--text-color);
+  background-color: var(--bg-color);
+  font-family: 'Comic Neue', cursive, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  text-decoration: none;
+}
+
+h1 {
+  font-size: 4rem;
+  font-weight: 700;
+}
+
+h2 {
+
```

Comment:
Why are these empty?

Reply:
will add exact sizes later, just a template for now